tests/test_utils.py

```diff
@@ -930,6 +930,51 @@ def test_read_csv(csv, result):
         pd.testing.assert_frame_equal(obj, result)
 
 
+@pytest.mark.parametrize(
+    'index, extension, expected_index',
+    [
+        (
+            audformat.filewise_index(),
+            'mp3',
+            audformat.filewise_index(),
+        ),
+        (
+            audformat.segmented_index(),
+            'mp3',
+            audformat.segmented_index(),
+        ),
+        (
+            audformat.filewise_index(['f1.wav', 'f2.wav']),
+            'mp3',
+            audformat.filewise_index(['f1.mp3', 'f2.mp3']),
+        ),
+        (
+            audformat.segmented_index(['f1.wav', 'f2.wav']),
+            'mp3',
+            audformat.segmented_index(['f1.mp3', 'f2.mp3']),
+        ),
+        (
+            audformat.filewise_index(['f1.WAV', 'f2.WAV']),
+            'MP3',
+            audformat.filewise_index(['f1.MP3', 'f2.MP3']),
+        ),
+        (
+            audformat.filewise_index(['f1', 'f2.wv']),
+            'mp3',
+            audformat.filewise_index(['f1', 'f2.mp3']),
```

Comment:
Here I'm not completely sure what should happen for a file name without extension. Should it also be changed to `f1.mp3`?

Reply:
> I did not call the function `replace_file_extension()` as we don't ask to replace a particular file extension

But that means I cannot replace `tar.gz` with `zip`, for instance.